edit-post/components/header/feature-toggle/index.js

```diff
@@ -9,30 +9,28 @@ import { withSelect, withDispatch } from '@wordpress/data';
 import { __ } from '@wordpress/i18n';
 import { compose } from '@wordpress/compose';
 import { MenuItem } from '@wordpress/components';
-import { ifViewportMatches } from '@wordpress/viewport';
 
-function FixedToolbarToggle( { onToggle, isActive } ) {
+function FeatureToggle( { onToggle, isActive, label } ) {
 	return (
 		<MenuItem
 			icon={ isActive && 'yes' }
 			isSelected={ isActive }
 			onClick={ onToggle }
 			role="menuitemcheckbox"
 		>
-			{ __( 'Fix Toolbar to Top' ) }
+			{ __( label ) }
```

Comment:
Minor: Should the internationalization of label be a responsibility of the user of FeatureToggle? In other components that seems to be the approach, we follow for labels.

Reply:
It isn't minor. It won't work properly this way.

Reply:
That's definitely a bug :P The label should already be localized at this point though :)